.gitignore

```diff
@@ -135,4 +135,7 @@ runtime/Cpp/runtime/libantlr4-runtime.dylib
 /runtime/Cpp/runtime/libantlr4-runtime.4.12.0.dylib
 
 # Go test and performance trace files
-**/*.pprof
\ No newline at end of file
+**/*.pprof
+runtime/Go/antlr/internal/testrig/antlr/*.jar
```

Comment:
Over specifying is a thing in .gitignore. If someone changed the global ignore for their own purposes, then this would still stay in force.

I have removed this internal directory to avoid confusion - I will keep it elsewhere.